src/handlers/canonicalFieldsHandlers.ts

```diff
@@ -1,13 +1,35 @@
+import { logger } from '../logger';
```

Comment:
I saw in #49 you used `.child` -- that's super interesting and I think speaks to the broader question I ask in another thread of "how do we want to do logging".

Whatever we decide, let's document it.

Reply:
Based on my familiarity with log4j/logback/slf4j etc., I was looking in pino for a way to qualify log messages by file. The child logger was the first, most obvious solution that I saw so I ran with it. If there is a more natural or better way to be able to filter by file, I am up for that too, so long as it doesn't require me to remember with each log message to add something there. If there is a more automated way to find the filename rather than a string literal, that would be cool, too, because then we can avoid the inevitable lies deriving from a copy/paste/fail-to-edit.

Reply:
Brilliant -- I'll implement that pattern in this PR for sure (and take a stab at making it more copy-paste friendly)

Reply:
Attempt has been made -- looking forward to any feedback! (marking this as resolved in the mean time)